docker/django/Dockerfile

```diff
@@ -56,6 +56,8 @@ FROM appbase AS production
 COPY --from=staticbuilder --chown=appuser:appuser /app/static /app/static
 COPY --chown=appuser:appuser . /app/
 
+RUN chgrp -R 0 /app/templates/rest_framework/ && chmod g+w -R /app/templates/rest_framework/
```

Comment:
All POST https://linkedevents-api-dev.agw.arodevtest.hel.fi/v1/image/ requests return 500 error and log shows `PermissionError: [Errno 13] Permission denied: '/app/media'` error

Would be nice to grant permissions to /app/media folder as well. e.g.
`RUN chgrp -R 0 /app/media/ && chmod g+w -R /app/media/`
See https://helsinkicity.slack.com/archives/C04REHSMNPQ/p1679303500397519

Maybe that could be done in scope of this PR as well?

Reply:
This will require a different fix. PR for that coming soon. :+1:

Reply:
Very good! Thanks